tests/testthat/test-DataDiscoveryRetrieval.R

```diff
@@ -112,3 +112,13 @@ test_that("TADAdataRetrieval", {
                     "MethodSpecificationName") %in% names(tada1)))
   
 })
+
+#testing that "meters" is successfully replaced with "m". This feature is part of the autoclean function
+#which runs automatically when TADAdataRetrieval runs
+test_that("TADAdataRetrieval", {
+  check_autoclean_meters_works <- TADAdataRetrieval(statecode = "UT",
+                                    characteristicName = c("Ammonia", "Nitrate", "Nitrogen"),
+                                    startDate = "01-01-2021")
+  expect_equal(check_autoclean_meters_works$ActivityDepthHeightMeasure.MeasureUnitCode[975], "m"
```

Comment:
If I understand this right you're running the function on the data retrieved by TADAdataRetrieval function. If data is ever added that will be included in this query result could the results at index 975 change? One way around that is to have the current result as a static file in the test data and run the function on that.

Reply:
Realized because this is testing autoclean and that is part of the retrieval process its a bit more complicated... so may have to just keep an eye out if this test fails that it may be an index change.

Reply:
Good points. USGS is also planning to actually change their data in the future (~summer 2023) or consistency with WQX, with includes things like using "m" instead of "meters". When that happen, it would would render this specific piece of code within autoclean and this test unnecessary --so in the future this can probably be deleted. 

This is all helpful to keep in mind so I'll try to track this discussion somewhere, maybe in a new issue. It might end up being helpful to track if an index changes on retrieved data in certain cases too, because that would mean the backend data profile or data itself is changing for some reason (e.g. could be a sign of updates to WQX 3.0 profiles or USGS actually changing their data in the future, etc.).